hknweb/events/models.py

```diff
@@ -1,66 +1,53 @@
 from django.db import models
+from hknweb.models import Profile
 
 class Event(models.Model):
-    id          = models.IntegerField(primary_key=True)
     name        = models.CharField(max_length=255, null=False)
     slug        = models.CharField(max_length=255)
     location    = models.CharField(max_length=255)
     description = models.TextField()
     start_time  = models.DateTimeField(null=False)
     end_time    = models.DateTimeField(null=False)
-    event_type_id = models.IntegerField()
+    rsvp_limit  = models.PositiveIntegerField(null=True, default=999999) # I'm being rather pessimistic in assuming no HKN event will hit 1 million participants!
```

Comment:
Since rsvp limits are optional, this might be better encoded as:
```py
rsvp_limit = models.PositiveIntegerField(null=True, blank=True)
```
That way `null=True` means the SQL value can be NULL, and the `blank=True` means in forms this field can be omitted. In particular, when the SQL value is NULL, Event.rsvp_limit will evaluate as `None` in Python (don't quote me, confirmation needed).

Source: https://docs.djangoproject.com/en/2.2/ref/models/fields/#null